Formula/s/swiftly.rb

```diff
@@ -30,26 +26,29 @@ class Swiftly < Formula
   end
 
   def install
-    system "swift", "build", "--configuration", "release", "--product", "swiftly", "--disable-sandbox"
+    system "swift", "build", "--configuration", "release", "--product", "swiftly", "--disable-sandbox" if OS.mac?
+
+    if OS.linux?
+      system "swift",
+             "build",
+             "-Xswiftc", "-I#{HOMEBREW_PREFIX}/include",
+             "-Xlinker", "-L#{HOMEBREW_PREFIX}/lib",
+             "--configuration", "release",
+             "--static-swift-stdlib",
+             "--product", "swiftly",
+             "--disable-sandbox"
+    end
+
     bin.install ".build/release/swiftly"
   end
 
   test do
+    # Test swiftly with a private installation
     swiftly_bin = testpath/"swiftly"/"bin"
     mkdir_p swiftly_bin
     ENV["SWIFTLY_HOME_DIR"] = testpath/"swiftly"
     ENV["SWIFTLY_BIN_DIR"] = swiftly_bin
-    system bin/"swiftly", "init", "--assume-yes", "--no-modify-profile"
-    system bin/"swiftly", "install", "latest", "--use"
-    (testpath/"main.swift").write <<~EOS
-      @main
-      struct HelloSwiftly {
-        static func main() {
-          print("Hello Swiftly!")
-        }
-      }
-    EOS
-    system swiftly_bin/"swiftc", "main.swift", "-parse-as-library"
```

Comment:
The test taking further step is due to [this thread](https://github.com/Homebrew/homebrew-core/pull/217092#discussion_r2059806769). Not sure if just the installation of the toolchain would be sufficient.

Reply:
It would be. The problem was with `--help` which rarely catches anything useful.